charts/zaakbrug/values.yaml

```diff
@@ -0,0 +1,146 @@
+replicaCount: 1
+
+image:
+  repository: wearefrank/zaakbrug
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podLabels: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: false
+  className: ""
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: chart-example.local
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+livenessProbe:
+  initialDelaySeconds: 40
+  periodSeconds: 10
+  timeoutSeconds: 1
+  failureThreshold: 6
+  successThreshold: 1
+
+readinessProbe:
+  initialDelaySeconds: 40
+  periodSeconds: 10
+  timeoutSeconds: 1
+  failureThreshold: 6
+  successThreshold: 1
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+autoscaling:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 100
+  targetCPUUtilizationPercentage: 80
+  # targetMemoryUtilizationPercentage: 80
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+# For database
+timeZone: Etc/UTC
+
+zaakbrug:
+  zgw: 
+    baseurl: 
+    endpoint: 
+      zaak:
+      zaaktype:
+      rol:
+      roltype:
+      statustype:
+      status:
+      resultaattype:
+      resultaat:
+      informatieobjecttype:
+      enkelvoudiginformatieobjecten:
+      zaakinformatieobjecten:
+
+frank:
+  # Secret containing the contents of the credentials.properties.
+  credentials: 
+    secret:
+    # Key of secret and place to mount.
+    key:
+  # (required) the name of the Frank
+  instance:
+    name: "ZaakBrug"
+  dtap:  
+    # (required) Options: LOC, DEV, TST, ACC, PRD
+    stage:
```

Comment:
Default DEV?

Reply:
I'm not sure. The thought process of leaving it empty is as follows:
A Frank! normally won't start if you haven't set the instance.name and dtap.stage. 
I thought to replicate this functionality by leaving it empty and to give a warning while installing the helm.
It forces the user to think about wich dtap stage should be used.